common/azure-common/src/main/java/com/azure/common/annotations/UnexpectedResponseExceptionTypes.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * The {@code @Repeatable} container annotation for {@link UnexpectedResponseExceptionType}. This allows methods to
+ * have different exceptions to be thrown or returned based on the response status codes returned from a REST API.
+ */
+@Retention(RUNTIME)
+@Target(METHOD)
+public @interface UnexpectedResponseExceptionTypes {
+    /**
+     * @return array of {@link UnexpectedResponseExceptionType} that annotate a method.
+     */
+    UnexpectedResponseExceptionType[] value();
+}
```

Comment:
I don't believe this annotation is required.

Reply:
This is required as the Repeatable annotation needs a class that has a value method that returns an array of the class its annotating, in this case `UnexpectedResponseExceptionType[] value()`.